docs/includes/apiargs-MongoDBCollection-method-createIndexes-option.yaml

```diff
@@ -1,3 +1,22 @@
+arg_name: option
+name: commitQuorum
+type: string|integer
+description: |
```

Comment:
Is there any way these can share the same include? Rather than add this to `apiargs-MongoDBCollection-common-option.yaml` perhaps we could just include the content from `apiargs-MongoDBCollection-method-createIndex-option.yaml`.